odxtools/parameters/createanyparameter.py

```diff
@@ -212,20 +221,23 @@ def create_any_parameter_from_et(et_element, doc_frags):
 
         parameter_id = OdxLinkId.from_et(et_element, doc_frags)
         table_ref = OdxLinkRef.from_et(et_element.find("TABLE-REF"), doc_frags)
-        table_snref = (
-            et_element.find("TABLE-SNREF").get("SHORT-NAME")
-            if et_element.find("TABLE-SNREF") is not None else None)
-        row_snref = (
-            et_element.find("TABLE-ROW-SNREF").get("SHORT-NAME")
-            if et_element.find("TABLE-ROW-SNREF") is not None else None)
-        row_ref = OdxLinkRef.from_et(et_element.find("TABLE-ROW-REF"), doc_frags)
+        if (table_snref_elem := et_element.find("TABLE-SNREF")) is not None:
+            table_snref = odxrequire(table_snref_elem.get("SHORT-NAME"))
+        else:
+            table_snref = None
+
+        table_row_ref = OdxLinkRef.from_et(et_element.find("TABLE-ROW-REF"), doc_frags)
+        if (table_row_snref_elem := et_element.find("TABLE-ROW-SNREF")) is not None:
+            table_row_snref = odxrequire(table_row_snref_elem.get("SHORT-NAME"))
+        else:
+            table_row_snref = None
```

Comment:
maybe a helper function to read SNREF?

Reply:
good point, but let's do this in another PR...